src/app/components/Curation/HierarchicalGrid/index.tsx

```diff
@@ -8,24 +8,43 @@ import VisuallyHiddenText from '../../VisuallyHiddenText';
 import formatDuration from '../../../lib/utilities/formatDuration';
 import Promo from '../../../legacy/components/Promo';
 import { DESKTOP, TABLET, MOBILE, SMALL } from './dataStructures';
-import { styles } from './index.styles';
+import {
+  CompactPromo,
+  HorizontalPromo,
+  SmallCompactPromo,
+  styles,
+  VerticalPromo,
+} from './index.styles';
 import { ServiceContext } from '../../../contexts/ServiceContext';
 import { CurationGridProps } from '../types';
 import { RequestContext } from '../../../contexts/RequestContext';
 
-const getStyles = (promoCount: number, i: number, mq: Theme['mq']) => {
+const getStyles = (
+  promoCount: number,
+  i: number,
+  mq: Theme['mq'],
+  visualProminence?: string,
+) => {
+  const smallStyle =
+    visualProminence === 'LOW' ? SmallCompactPromo : SMALL[promoCount - 1][i];
+  const mobileStyle =
+    visualProminence === 'LOW' ? SmallCompactPromo : MOBILE[promoCount - 1][i];
+  const tabletStyle =
+    visualProminence === 'LOW' ? SmallCompactPromo : TABLET[promoCount - 1][i];
+  const desktopStyle =
+    visualProminence === 'LOW' ? SmallCompactPromo : DESKTOP[promoCount - 1][i];
```

Comment:
Can we leave the tablet & desktop styles as they were, for demoing to the UX team? 
```suggestion
  const tabletStyle = TABLET[promoCount - 1][i];
  const desktopStyle = DESKTOP[promoCount - 1][i];
```